web/modules/custom/joinup_core/src/EventSubscriber/RefreshCachedFieldsEventSubscriber.php

```diff
@@ -303,11 +353,11 @@ protected function getFileUrl(ContentEntityInterface $entity) {
    * @param string $bundle
    *   The bundle for which to retrieve the settings.
    *
-   * @return array|false
+   * @return array
    *   The configuration array, or FALSE if there is no configuration for this
```

Comment:
Here also the return type is updated correctly but the documentation still mentions that it can return `FALSE`. It will always return an array, it just might be empty.